From b09a0ce08f376e2505b84368b65b0583f145fb6d Mon Sep 17 00:00:00 2001
From: Jason Chen <b02280@freescale.com>
Date: Mon, 8 Feb 2010 15:24:12 +0800
Subject: [PATCH 099/259] ENGR00120840 ipu lib: COPY_MODE for internal memory
 mode

If there is nothing need to process, enable COPY_MODE.

Signed-off-by: Jason Chen <b02280@freescale.com>
---
 ipu/mxc_ipu_hl_lib.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/ipu/mxc_ipu_hl_lib.c b/ipu/mxc_ipu_hl_lib.c
index 32dd76e..13d9ff3 100644
--- a/ipu/mxc_ipu_hl_lib.c
+++ b/ipu/mxc_ipu_hl_lib.c
@@ -724,16 +724,9 @@ static int _ipu_task_check(ipu_lib_input_param_t * input,
 	} else {
 		/* 1 output case */
 		if (ipu_priv_handle->output[0].task_mode == NULL_MODE) {
-			if (input->user_def_paddr[0] && output0->user_def_paddr[0]) {
 				ipu_priv_handle->output[0].task_mode = COPY_MODE;
 				dbg(DBG_INFO, "Copy case!\n");
 				goto done;
-
-			} else {
-				dbg(DBG_ERR, "Do not need any operation! Will do nothing!\n");
-				ret = -1;
-				goto done;
-			}
 		}
 
 		/* try ENC first */
-- 
1.8.0

